### PR TITLE
Fix division by zero

### PR DESCRIPTION
**Pull request by tgflynn**
Fix divide by zero bug on startup

Ensure load testing code does not run on mainnet

### CODE REVIEW

File: src/governance.cpp [hunk 1]
```diff
@@ -997,8 +997,11 @@ void CGovernanceManager::RequestGovernanceObjectVotes(const std::vector<CNode*>&
     // so 1 obj on mainnet == ~10 objs or ~1000 votes on testnet. However we want to test a higher
     // number of votes to make sure it's robust enough, so aim at 2000 votes per masternode per request.
     // On mainnet we have 4K+ masternodes, so nMaxObjRequestsPerNode always evaluates to `1`.
-    size_t nProjectedVotes = 2000;
-    int nMaxObjRequestsPerNode = std::max(1, int(nProjectedVotes / mnodeman.size()));
+    size_t nProjectedVotes = 1;
```
Comment (UdjinM6) [inline src/governance.cpp]: I would keep `nProjectedVotes` as is but set `nMaxObjRequestsPerNode = 1` for mainnet.
```
size_t nProjectedVotes = 2000;
int nMaxObjRequestsPerNode = 1;
if(Params().NetworkIDString() != CBaseChainParams::MAIN) {
    nMaxObjRequestsPerNode = std::max(1, int(nProjectedVotes / std::max(1, mnodeman.size())));
}

Comment (UdjinM6) [inline src/governance.cpp]: Maybe also fix comment above:
```
// On mainnet nMaxObjRequestsPerNode is always equal to `1`.
```
or smth like that

Comment (tgflynn) [inline src/governance.cpp]: OK, but we also need to change nProjectedVotes because it's used in code below.

Comment (UdjinM6) [inline src/governance.cpp]: yes, it's used in code below but just as some kind of a threshold for `pnode->setAskFor.size()` and 2000 seems reasonable for mainnet too.

Comment (tgflynn) [inline src/governance.cpp]: It seems to me that for mainnet setAskFor size should only increase by 1 per iteration, shouldn't it ?

Comment (UdjinM6) [inline src/governance.cpp]: No, the point here is this: we try to predict how many votes we are going to get if we request an obj (i.e. nProjectedVotes) and if that would make setAskFor size to grow over some some threshold we don't even try to ask, we skip to another node.

**Pull request merged**